exports.some_function = async function(){
	try {
		let value = await promise()
	}catch (exception){

	}
}